tutorials/12_CES_Calibration.md

```diff
@@ -0,0 +1,225 @@
+
+The calibration can further be adjusted using three switches:
+
+- `c_CES_calibration_iterations`: The number of CES calibration iterations
+  (iterative adjustments of $\alpha_i$ parameters, see above).  By default ten.
+  We don't use a convergence criterion but check the calibration quality manual
+  afterwards.  Higher iteration numbers will generally lead to better
+  convergence, but at a runtime expense, as each iteration is an additional
+  REMIND run.
+- `c_CES_calibration_new_structure`: Switch to use a default price for the
+  primary production factors instead of those derived from a .gdx file.  Only
+  necessary when the structure of the CES tree was changed, but can be used to
+  manually force specific prices in the first CES calibration iteration.  The
+  calibration will abort if this switch is necessary but unset, so it should not
+  be used unless it is known to be necessary.
+- `cm_CES_calibration_default_prices`: Default price to be used when
+  `c_CES_calibration_new_structure` is set.  By default 0.01.  The price is not
+  differentiated by period, region, or production factor.  A price far off from
+  the equilibrium prices will lead to longer calibration times/less good
+  calibration performance for a given number of iterations.  Too high prices can
+  lead to errors during the calibration.
+
+Furthermore, there is the option `c_CES_calibration_write_prices`, which forces
+the prices calculated as partial derivatives on the CES production function to
+be written to the file `pm_cesdata_price`, which might be helpful in finding a
+suitable value for `cm_CES_calibration_default_price`, but is generally not
+used.
+
+
+## Results
+
+The main output of the CES calibration are a .gdx file and a .inc files with all
+the CES parameters.  They are named based on the CES configuration, the
+GDP/population scenarios, the capital market module realisation and the region
+configuration (e.g.
+`indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2EU-GDP_gdp_SSP2EU-En_gdp_SSP2EU-Kap_debt_limit-Reg_62eff8f7`).
+You don't need to change these names, they are matched automatically.  The
+parameter files also include a counter for the calibration iteration they
+resulted from (e.g. `_ITERATION_10.inc`).  To use the calibration results, copy
+the parameter (`.inc`) file (without the iteration counter) to the directory
+`./modules/29_CES_parameters/load/input/` and the .gdx file to the directory
+`./config/gdx-files/`.  At PIK, this is done automatically using the RSE support
+scripts.  See [this wiki
+page](https://redmine.pik-potsdam.de/projects/remind-r/wiki/GDX_and_CES_parameter_Handling)
+for details.
+
+As for diagnostic output, there are the `full.log` and `full.lst` files for each
+calibration iteration (`full_01.log` …), the file `CES_calibration.csv`
+containing all the relevant calibration parameters (inputs and outputs) for all
+iterations for automated analysis and a `CES_calibration_report` .pdf file with
```

Comment:
add info that the `CES_calibration_report` also can be generates by "Rscript output.R" up to any existing CES-iteration afterwards